roles/kube_elasticsearch/tasks/main.yml

```diff
@@ -21,6 +22,26 @@
   run_once: true
   delegate_to: "{{ groups['kube-master'][0] }}"
 
+- name: 'Delete old es-client deployment'
```

Comment:
At the end (define: end ?). We want to manage migration from point to point version of from any revision to any revision ? Thos code for example has little chance to be triggered in production (as we have no production). We should discuss this also for a CI point a view.